damnit/gui/table.py

```diff
@@ -176,6 +184,34 @@ def get_static_columns(self):
     def get_static_columns_count(self):
         return self._static_columns_widget.count()
 
+    #Copied from https://stackoverflow.com/questions/22635867/is-it-possible-to-set-the-text-of-the-qtableview-corner-button
+    #to suport the run Verticalheader's header
+    def eventFilter(self, obj, event):
+        if event.type() != QtCore.QEvent.Paint or not isinstance(
+                obj, QtWidgets.QAbstractButton):
+            return False
+
+        # Paint by hand (borrowed from QTableCornerButton)
+        opt = QtWidgets.QStyleOptionHeader()
+        opt.initFrom(obj)
+        styleState = QtWidgets.QStyle.State_None
+        if obj.isEnabled():
+            styleState |= QtWidgets.QStyle.State_Enabled
+        if obj.isActiveWindow():
+            styleState |= QtWidgets.QStyle.State_Active
+        if obj.isDown():
+            styleState |= QtWidgets.QStyle.State_Sunken
+        opt.state = styleState
+        opt.rect = obj.rect()
+        # This line is the only difference to QTableCornerButton
+        opt.text = obj.text()
+        opt.position = QtWidgets.QStyleOptionHeader.OnlyOneSection
+        painter = QtWidgets.QStylePainter(obj)
+        painter.drawControl(QtWidgets.QStyle.CE_Header, opt)
+
+        return True
```

Comment:
This is... clever, but it's a lot of low-level fiddling to make the table headers look almost, but not exactly, like a regular column. 

![image](https://github.com/European-XFEL/DAMNIT/assets/327925/69b2fb18-4ff4-48f3-9fb8-47c489c1d100)

I'd rather just leave the corner button blank, rely on people figuring out that the numbers are run numbers, and get rid of this code.